_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,191 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope), except if the element has an implicit semantic role that is identical to its explicit semantic role.
+
+### Expectation
+
+The [owner element](owner-element) for each target element has a [semantic role](#semantic-role) matching one of the [WAI-ARIA required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) for the target element.
+
+## Assumptions
```

Comment:
Below are two examples of html without a required role context that we would approve under WCAG. 
I think the following assumption should be added:' The aria-role is being used with the intention to comply to WCAG'.

```
<h1>Information tabs</h1>
            <div role="tab">
                <h2>tab1</h2>
            </div>
            <p>This text belongs to tab1</p>
            <div role="tab">
                <h2>tab2</h2>
            </div>
            <p>This text belongs to tab2</p>
            <div role="tab">
                <h2>tab3</h2>
            </div>
            <p>This text belongs to tab3</p>
        <h1>menu</h1>
            <ul>
                <li>
                    <div role="menuitem"><a>Menu item 1</a></div>
                </li>
                <li>
                    <div role="menuitem"><a>Menu item 2</a></div>
                </li>
                <li>
                    <div role="menuitem"><a>Menu item 3</a></div>
                </li>
            </ul>
```

Reply:
@danistr, can you elaborate on why you would pass these? I am not sure I follow...
And in which cases would you then fail something for this?

Reply:
Succescritirium 1.3.1 demands that relationships and structures are communicated. In the above examples the relationship between headings and paragraphs and the relationships between different menu items is already communicated without aria. Some screenreaders might mention the fact that these are menu items or tabs which would elaborate on the structure of the page but would not be necessary to comply to WCAG. In short: we would also pass these examples if they had no aria at all, and the aria that is added doesn’t communicate anything in error.

We would fail something for this if the structure or relationship was not communicated apart from with aria and aria was not used correctly. For example:

```html
<div role="listitem">Apple</div>
<div role="listitem">Banana</div>
<div role="listitem">Kiwi</div>
```
The list above might not be communicated to a screenreader user and would fail WCAG.

Reply:
@danistr, thanks for the explanation. 
I will add an assumption.

Reply:
That's a very letter-of-the-law interpretation @danistr. I can see my legacy at a11yNL lives on :)

Reply:
@danistr, I have added an assumption. Does it capture your thoughts about this?

Reply:
> @danistr, I have added an assumption. Does it capture your thoughts about this?

Yes, the added assumption is what i mean. Looks good :)